src/main/java/com/github/psambit9791/jdsp/filter/FIRWin1.java

```diff
@@ -40,6 +40,15 @@
     private double width;
     private double nyquistF;
 
+    public enum FIRfilterType {
```

Comment:
Any particular reason for this? It would add an extra datatype that the user has to initialise. The string seemed neater to me.

Reply:
You need to make your programs as foolproof as possible. Doing it this way eliminates all the times that a user would misspell the String filter type. Using an enum is also just a go-to way of implementing type selection and from a programmer perspective a cleaner implementation.